content/documentation/device-examples/components/Camera.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Camera
+weight: 210
+tags: ["Camera"]
+---
+### Description
+The [Camera](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/components/components) (src/main/java/com/pi4j/components/components) is a template class, that you can use in your own Java-project.
```

Comment:
do not use Dev-Arcade link, use the link from the main branch. I have corrected this in all other fiels

Reply:
corrected